doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -214,8 +214,18 @@ Before you start with C application code, some preparation is required.
 
 <div id="prep-obj-c" class="tab-pane fade" markdown="1" >
 
-</div>
+You have to make some preparations before start with Objective-C application code.
+
+1. If you haven't done, please install CocoaPods.
+2. Extract Kaa SDK, open it folder in terminal and run
 
+        sh build.sh compile
+3. Go to xCode and choose template of the iOS Single View Application. Name it "My First Kaa Application", choose Objective-C language and leave other fields default values.
+4. Link SDK to your project as described in <a href="http://kaaproject.github.io/kaa/doc/client-objective-c/latest/index.html?src=contextnav">Objective-C SDK Reference</a>.
+5. Make sure that your application builds successfully.
+6. Replace code of `ViewController.m` with code from [Application code](#application-code) section.
+
```

Comment:
Before using the Objective-C application code, be sure to complete the following steps:

1. Install CocoaPods.
2. Extract Kaa SDK, open a terminal from the extraction directory, and run the following command.

        sh build.sh compile
3. Go to xCode and choose a template of the iOS Single View Application.
Name it "My First Kaa Application", choose Objective-C language and leave all other fields unchanged.
4. Link the SDK to your project as described in [Objective-C SDK Reference]({{root_url}}client-objective-c/latest/index.html?src=contextnav").
5. Make sure that your application builds successfully.
6. Replace code in the ViewController.m file with the code from the [Application code](#application-code) section.